### ISSUE TITLE
Inconsistent field name: title vs role

### ISSUE THREAD

**Issue by konklone** [propublica/congress-api-docs#43]
On the [state/district member endpoint](https://projects.propublica.org/api-docs/congress-api/endpoints/#get-current-members-by-statedistrict), it uses `role` where other endpoints for members use `title`. It would be nice if these were consistent.

**Comment by konklone** [comment]
To broaden this a bit: the `Lists of members` endpoint seems to have neither a `title` nor a `role` field:

https://projects.propublica.org/api-docs/congress-api/endpoints/#lists-of-members

While one may have the chamber on-hand in the client (because it was made a URL parameter), this isn't sufficient to distinguish Representatives from Delegates from Resident Commissioners.

**Comment by jalefkowit** [comment]
Yeah, the inability to filter out Delegates/RCs/etc. from Representatives is biting me hard on a project I'm working on. With the old Sunlight API it was possible to look at the title field in the results to make that happen. Not sure what the value is of not having the lists-of-members endpoint include it.

**Comment by dwillis** [comment]
I think we can add it, but in terms of filtering those folks out, you can do so by `state`, excluding the following values: PR, DC, VI, MP, GU, AS.
